src/notifications.c

```diff
@@ -17,9 +17,13 @@
 #include "dictionary.h"
 #include "slot_ranges.h"
 #include "asm_state_machine.h"
+//TODO ASM: Review this, maybe is worth duplicating the logic or moving to utils?
+#include "src/coord/rmr/redis_cluster.h"

```

Comment:
I think it make sense. The notification unit in the current state of the code can rely on (many) other compnents, while there should be very small amount of compnents relying on the notification compnent.